lookout/core/lib.py

```diff
@@ -9,21 +9,29 @@
 from lookout.core.api.service_data_pb2 import File
 
 
-def find_new_lines(before: File, after: File) -> List[int]:
+def find_modified_lines(before: File, after: File) -> List[int]:
```

Comment:
So there is a problem with terminology here.

`find_new_lines` returns the lines which are either added or changed. IT should forever stay so.

What we should really do is to add `find_deleted_lines` which returns the deleted line indices **in the new file coordinates**. This is something not every analyzer needs, so our code will
```
sorted(chain.from_iterable(find_new_lines(), find_deleted_lines()))
```
which is equvalent to the coded `find_modified_lines`

Reply:
ok

Reply:
one thing, it requires to run `SequenceMatcher` two times and it is kind of heavy operation. Are we fine with that?

Reply:
It's not heavy compared to getting UASTs and especially evaluating our rules.